command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
```

Comment:
I do not think that user banning necessitates dedicated commands. I would rather see them possible through `account update`.

Reply:
If we do this, is there anything keeping the attributes separate from `update` as well? I find the distinction sort of useless.

Reply:
Yes, there is a key difference. `account attribute` is like `rse attribute` and `did metadata`. It is a key-value storage of arbitrary data. The command-line parameters will be `--key` and `--value` and any content set to them is fine. Instead, `account/rse/did update` will have specific options that may of may not take parameter (e.g. for replication rules `--lifetime=0` and `--stuck`). And there is potentially additional processing on the user input (e.g. `--lifetime=0` sets it to now).

Reply:
I think ideally that's true, but as it stands (for account specifically), the `account update` just takes a key value pair, identically to the `account attribute add`. Is there anything you can do to update an account besides banning that should be included in the `update` functionality? (Or is that a separate issue).

Reply:
Currently it’s only the `email` column.But there is a difference between `account update` (update the column in the `accounts` table) and `account attribute add` (add a new row in the `account_attr_map` table). And who knows whether we might need more columns in the `accounts` table in the future?

Reply:
Ah, I misunderstood where the email field was being stored. I understand the distinction then, and agree with your original suggestion.